ci/dra_upload.sh

```diff
@@ -7,7 +7,7 @@
 export JRUBY_OPTS="-J-Xmx1g"
 
 STACK_VERSION=`cat versions.yml | sed -n 's/^logstash\:\s\([[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*\)$/\1/p'`
-RELEASE_BRANCH=`git rev-parse --abbrev-ref HEAD`
+RELEASE_BRANCH=`cat versions.yml | sed -n 's/^logstash\:\s\([[:digit:]]*\.[[:digit:]]*\)\.[[:digit:]]*$/\1/p'`
```

Comment:
```suggestion
RELEASE_BRANCH=`cat versions.yml | sed -n 's/^logstash\:[[:space:]]\([[:digit:]]*\.[[:digit:]]*\)\.[[:digit:]]*$/\1/p'`
```
Can we use [[:space:]] instead of \s to make it works in Mac as well?

Reply:
which is the difference, in matching terms, between `\s` and `[:space:]` ?

Reply:
`[[:space:]]` is POSIX compliance. `\s` matches tab and space, and Mac fails to take the version with \s

Reply:
Thanks for clarifying :-) :+1